### PR TITLE
modulesync 1.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
I've no idea why this fails :(

**Comment by domcleal** [comment]
Primarily this removal:

* https://github.com/voxpupuli/puppet-puppetserver/pull/46/files#diff-93830fa29d616f7c87903d08b5b1b29aL5 (Ruby load path modifications in spec_helper.rb)

but these are also required:

* https://github.com/voxpupuli/puppet-puppetserver/pull/46/files#diff-354f30a63fb0907d4ad57269548329e3L3 (libaugeas-dev in travis.yml)
* https://github.com/voxpupuli/puppet-puppetserver/pull/46/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L38 (ruby-augeas in Gemfile)

They need adding to .sync.yml (and possibly modulesync_config templates).

**Comment by domcleal** [comment]
I extracted the spec_helper.rb hacks into https://github.com/voxpupuli/modulesync_config/pull/321, so on the next msync config release it should sync correctly (the flag is already in .sync.yml, probably from the module's previous owners).

The missing gem and package are fixed now in this branch.

**Comment by wyardley** [comment]
this also needed a rebase I think?

**Comment by wyardley** [comment]
@bastelfreak Also fixed some remaining Rubocop failures.
Adding augeas in the Gemfile does make it a bit harder to run the spec tests, as the installation fails with a cryptic error if augeas itself isn't installed on the system running it.

File: spec/defines/puppetserver__config__java_arg_spec.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
         context 'when not declaring Class[puppetserver]' do
           it 'fails' do
-            expect { is_expected.to compile }.to raise_error(%r{Could not find resource 'Class\[Puppetserver::Install\]' for relationship on 'Puppetserver::Config::Java_arg\[foo\]'})
+            is_expected.to raise_error(%r{Could not find resource 'Class\[Puppetserver::Install\]' for relationship on 'Puppetserver::Config::Java_arg\[foo\]'})
```
Comment (wyardley) [inline spec/defines/puppetserver__config__java_arg_spec.rb]: I _think_ this does the same thing?

**Comment by wyardley** [comment]
This is green now, just need to get some clarification about the test discussed above.

**Pull request merged**